flake.nix

```diff
@@ -20,7 +20,14 @@
       packages = forAllSystems (
         { nixPackages, ... }:
         {
-          test = nixPackages.hello;
+          nodejs = rec {
+            v18 = nixPackages.callPackage ./runtimes/nodejs/v18.nix { };
+            v20 = nixPackages.callPackage ./runtimes/nodejs/v20.nix { };
+            v22 = nixPackages.callPackage ./runtimes/nodejs/v22.nix { };
```

Comment:
shall we try using `packagesFromDirectoryRecursive`? https://github.com/auxolotl/core/blob/cbe7a41cc89ccda6bfd14da7c2a8cf982ca07d24/lib/filesystem.nix#L189

Reply:
Perhaps. I'm not immediately seeing a particularly pretty way to handle `lts` and `latest` aliases with that, though.

Reply:
Actually - idea. Lemme push something.

Reply:
Just pushed something a little more refined, how do we feel about this?